crates/node/core/src/args/engine.rs

```diff
@@ -61,9 +61,13 @@ pub struct EngineArgs {
     pub reserved_cpu_cores: usize,
 
     /// Enable precompile cache
-    #[arg(long = "engine.precompile-cache", default_value = "false")]
+    #[arg(long = "engine.precompile-cache", default_value = "true", hide = true)]
     pub precompile_cache_enabled: bool,
```

Comment:
can we mark this as deprecated so we can remove this soon

Reply:
you mean mark the struct field as deprecated?

Reply:
then let's just add to docs